blockchain/tx_pool_test.go

```diff
@@ -40,13 +42,24 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-// testTxPoolConfig is a transaction pool configuration without stateful disk
-// sideeffects used during testing.
-var testTxPoolConfig TxPoolConfig
+var (
+	// testTxPoolConfig is a transaction pool configuration without stateful disk
+	// sideeffects used during testing.
+	testTxPoolConfig TxPoolConfig
+
+	// eip1559Config is a chain config with EIP-1559 enabled at block 0.
+	eip1559Config *params.ChainConfig
+)
 
 func init() {
 	testTxPoolConfig = DefaultTxPoolConfig
 	testTxPoolConfig.Journal = ""
+
+	cpy := *params.TestChainConfig
+	eip1559Config = &cpy
+	eip1559Config.IstanbulCompatibleBlock = common.Big0
+	eip1559Config.LondonCompatibleBlock = common.Big0
+	fork.SetHardForkBlockNumberConfig(eip1559Config)
```

Comment:
Isn't it overwriting `params.TestChainConfig` ?

Reply:
It don't affected `param.TestChainConfing` values. It just copy a value.

Reply:
Okay. I think it's okay because `IstanbulCompatibleBlock` and `LondonCompatibleBlock` are the first depth field of chainConfig. So, it doesn't need deep copy here